XenAdmin/Wizards/NewCPUGroupWizard/NewCPUGroupCPUCoresPage.Designer.cs

```diff
@@ -0,0 +1,175 @@
+            this.dataGridView1.ColumnHeadersBorderStyle = System.Windows.Forms.DataGridViewHeaderBorderStyle.None;
+            this.dataGridView1.ColumnHeadersHeightSizeMode = System.Windows.Forms.DataGridViewColumnHeadersHeightSizeMode.AutoSize;
+            this.dataGridView1.Columns.AddRange(new System.Windows.Forms.DataGridViewColumn[] {
+            this.CheckBoxColumn,
+            this.NameColumn,
+            this.DescriptionColumn,
+            this.CurrentCPUGroupColumn});
+            this.tableLayoutPanel1.SetColumnSpan(this.dataGridView1, 3);
+            this.dataGridView1.EditMode = System.Windows.Forms.DataGridViewEditMode.EditProgrammatically;
+            this.dataGridView1.GridColor = System.Drawing.SystemColors.Window;
+            this.dataGridView1.MultiSelect = false;
+            this.dataGridView1.Name = "dataGridView1";
+            this.dataGridView1.ReadOnly = true;
+            this.dataGridView1.RowHeadersVisible = false;
+            this.dataGridView1.SelectionMode = System.Windows.Forms.DataGridViewSelectionMode.FullRowSelect;
+            this.dataGridView1.ShowCellErrors = false;
+            this.dataGridView1.ShowEditingIcon = false;
+            this.dataGridView1.ShowRowErrors = false;
+            this.dataGridView1.StandardTab = true;
+            // 
+            // CheckBoxColumn
+            // 
+            resources.ApplyResources(this.CheckBoxColumn, "CheckBoxColumn");
+            this.CheckBoxColumn.Name = "CheckBoxColumn";
+            this.CheckBoxColumn.ReadOnly = true;
+            // 
+            // NameColumn
+            // 
+            resources.ApplyResources(this.NameColumn, "NameColumn");
+            this.NameColumn.Name = "NameColumn";
+            this.NameColumn.ReadOnly = true;
+            // 
+            // DescriptionColumn
+            // 
+            resources.ApplyResources(this.DescriptionColumn, "DescriptionColumn");
+            this.DescriptionColumn.Name = "DescriptionColumn";
+            this.DescriptionColumn.ReadOnly = true;
+            // 
+            // CurrentCPUGroupColumn
+            // 
+            resources.ApplyResources(this.CurrentCPUGroupColumn, "CurrentCPUGroupColumn");
+            this.CurrentCPUGroupColumn.Name = "CurrentCPUGroupColumn";
+            this.CurrentCPUGroupColumn.ReadOnly = true;
+            // 
+            // NewCPUGroupCPUCoresPage
+            // 
+            resources.ApplyResources(this, "$this");
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Dpi;
+            this.Controls.Add(this.tableLayoutPanel1);
+            this.Name = "NewCPUGroupCPUCoresPage";
+            this.tableLayoutPanel1.ResumeLayout(false);
+            this.tableLayoutPanel1.PerformLayout();
+            ((System.ComponentModel.ISupportInitialize)(this.dataGridView1)).EndInit();
+            this.ResumeLayout(false);
+
+        }
+
+        #endregion
+
+        protected System.Windows.Forms.TableLayoutPanel tableLayoutPanel1;

```

Comment:
The column containing the search box should be autosized (although I admit this is not consistent throughout XenCenter).

Reply:
I agree with you, thanks.